### PR TITLE
added ability to run application with docker

### PR DESCRIPTION
**Pull request by OlehSarapuk**

### CODE REVIEW

**Comment by OlehSarapuk** [comment]
@ajax-gnatiuk-s review please 😁

**Comment by ajax-gnatiuk-s** [comment]
@YuliaShvets please review

**Review by YuliaShvets — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  mongo:
+    image: mongo:latest
+    restart: unless-stopped
+    env_file: ./.env
+    ports:
+      - ${MONGO_LOCAL_PORT}:${MONGO_DOCKER_PORT}
+  app:
+    depends_on:
+      - mongo
+    image: ajax-onboarding-project
```
Comment (ajax-gnatiuk-s) [inline docker-compose.yml]: you can set path to `Dockerfile`. So you don't need to build the image manually before starting the docker-compose.

see https://docs.docker.com/compose/compose-file/build/#dockerfile

**Review by ajax-gnatiuk-s — COMMENTED** [review]
It's not clear why:
- dependencies were downgraded
- removed `gradle.properties` 
- removed gradle wrapper
- downgraded gradle version 8.3 -> 8.2
- removed plugin `org.gradle.toolchains.foojay-resolver-convention`

**Comment by OlehSarapuk** [comment]
Downgraded kotlin plugin version because of compatibility with detekt
Removed plugin org.gradle.toolchains.foojay-resolver-convention because there was no need in it

File: docker-compose.yml (hunk 1 above)
Comment (OlehSarapuk) [reply docker-compose.yml]: Done!

**Review by ajax-gnatiuk-s — APPROVED** [review]

**Comment by ajax-gnatiuk-s** [comment]
lgtm

**Pull request merged**